build.cmd

```diff
@@ -63,12 +66,11 @@ setlocal
     exit /b 1
   )
 
-  dotnet.exe publish -c %BuildConfiguration% --framework netcoreapp1.0  || exit /b 1
-  dotnet.exe publish -c %BuildConfiguration% --framework netcoreapp1.1  || exit /b 1
+  for %%v in (1.0 1.1 2.0) do (
```

Comment:
Just confirming - the netcoreappXX TFM is just for building simpleharness.  The APIs themselves are still netstandard, right?

Reply:
Right. I just wanted to increase our coverage now that 2.0 is out.